retrieval_service/datastore/providers/spanner_postgres.py

```diff
@@ -0,0 +1,1020 @@
+                    "p3": param_types.STRING,
+                    "p4": param_types.STRING,
+                },
+            )
+
+        if results is None:
+            return None
+
+        flights = [
+            models.Flight.model_validate(
+                {key: value for key, value in zip(self.FLIGHTS_COLUMNS, a)}
+            )
+            for a in results
+        ]
+        return flights[0]
+
+    async def insert_ticket(
+        self,
+        user_id: str,
+        user_name: str,
+        user_email: str,
+        airline: str,
+        flight_number: str,
+        departure_airport: str,
+        arrival_airport: str,
+        departure_time: str,
+        arrival_time: str,
+    ):
+        """
+        Inserts a ticket into the database.
+
+        Args:
+            user_id (str): The ID of the user.
+            user_name (str): The name of the user.
+            user_email (str): The email of the user.
+            airline (str): The airline of the flight.
+            flight_number (str): The flight number.
+            departure_airport (str): The departure airport code.
+            arrival_airport (str): The arrival airport code.
+            departure_time (str): The departure time of the flight.
+            arrival_time (str): The arrival time of the flight.
+        """
+        departure_time_datetime = datetime.datetime.strptime(
+            departure_time, "%Y-%m-%d %H:%M:%S"
+        )
+        arrival_time_datetime = datetime.datetime.strptime(
+            arrival_time, "%Y-%m-%d %H:%M:%S"
+        )
+
+        with self.__database.batch() as batch:
+            batch.insert(
+                table="tickets",
+                columns=[
+                    "user_id",
+                    "user_name",
+                    "user_email",
+                    "airline",
+                    "flight_number",
+                    "departure_airport",
+                    "arrival_airport",
+                    "departure_time",
+                    "arrival_time",
+                ],
+                values=[
+                    [
+                        user_id,
+                        user_name,
+                        user_email,
+                        airline,
+                        flight_number,
+                        departure_airport,
+                        arrival_airport,
+                        departure_time_datetime,
+                        arrival_time_datetime,
+                    ]
+                ],
+            )
+
+    async def list_tickets(
+        self,
+        user_id: str,
+    ) -> list[models.Ticket]:
+        """
+        Retrieves a list of tickets for a user.
+
+        Args:
+            user_id (str): The ID of the user.
+        """
+        with self.__database.snapshot() as snapshot:
+            # Spread SQL query for readability
+            results = snapshot.execute_sql(
+                sql="""
+                SELECT * FROM tickets
```

Comment:
We have a new changes for this to return only specific field instead of `*`. Please update this. Thank you!